### ISSUE TITLE
node render does not support custom images

### ISSUE THREAD

**Issue by lordnynex** [blushft/go-diagrams#9]
Small usability issue. I'm attempting to render a diagram that uses the GCP Pubsub icon (which the asset package does not appear to have). 
![pubsub](https://user-images.githubusercontent.com/1108594/94499596-f9664380-01b1-11eb-8fe9-2822c22fb2d2.png)

I thought I might be able to `diagram.NewNode(diagram.Icon("pubsub.png")).Label("QueueName")` but I noticed that [node render](https://github.com/blushft/go-diagrams/blob/5a1c57dcd0fb380430fb01edf96f2b084589091b/diagram/node.go#L46) expects the image path to be in the asset box. Would it be possible to fall back to local disk? Or maybe better yet provide a `NodeOption` like `CustomImage` to support custom icons?

**Comment by tntobias** [comment]
Good catch! We should absolutely find a way intuitively use images from outside the library. I have an idea on how to do this but wanna release an update before I tackle it. Thanks for calling attention to this.

**Comment by tntobias** [comment]
Also, the PubSub icon is under gcp.Analytics.Pubsub()

**Comment by Juandavi1** [comment]
hi, i have the same issue

**Comment by rvoh-emccaleb** [comment]
Any updates on this? Just ran into the issue, myself
